src/Servers/Kestrel/Core/src/LocalhostListenOptions.cs

```diff
@@ -41,7 +41,7 @@ internal override async Task BindAsync(AddressBindContext context)
             }
             catch (Exception ex) when (!(ex is IOException))
             {
-                context.Logger.LogWarning(0, CoreStrings.NetworkInterfaceBindingFailed, GetDisplayName(), "IPv4 loopback", ex.Message);
+                context.Logger.LogInformation(0, CoreStrings.NetworkInterfaceBindingFailed, GetDisplayName(), "IPv4 loopback", ex.Message);
```

Comment:
I believe we need to do the second part in this PR:
> make sure that if both fail there is an error log and exception.

So if both of these fail, can you add an exception with a LogError?

Reply:
That case is handled on line 59.

Our general guidance is to log _or_ throw, not both.